release_docs/RELEASE.txt

```diff
@@ -301,6 +301,14 @@ Bug Fixes since HDF5-1.13.3 release
 
     Configuration
     -------------
+    - Change the default for building the high-level tools
+
+      The gif2hdf5, hdf2gif and h5watch high-level tools are deprecated and will be removed
```

Comment:
Is h5watch really deprecated?

Reply:
All three tools get disabled.

Reply:
I could split the line to make h5watch in a sentence by itself. A bit confusing though.